plugins/intrinsics/src/main/java/org/qbicc/plugin/intrinsics/core/CoreIntrinsics.java

```diff
@@ -549,6 +645,11 @@ public static void registerJavaLangObjectIntrinsics(CompilationContext ctxt) {
         };
 
         intrinsics.registerIntrinsic(objDesc, "clone", MethodDescriptor.synthesize(classContext, objDesc, List.of()), clone);
+
+        // stub - public final native void wait(long timeoutMillis)
+        MethodDescriptor waitDesc = MethodDescriptor.synthesize(classContext, BaseTypeDescriptor.V, List.of(BaseTypeDescriptor.J));
+        InstanceIntrinsic wait = (builder, instance, target, arguments) -> ctxt.getLiteralFactory().zeroInitializerLiteralOfType(target.getType().getReturnType());
+        intrinsics.registerIntrinsic(objDesc, "wait", waitDesc, wait);
```

Comment:
This is OK temporarily (see above) but this method should be implemented directly in `java.lang.Object` in `qbicc-class-library` once we have monitors sorted out.